processor/processor.go

```diff
@@ -187,29 +196,30 @@ func (proc *HandleT) addJobsToSessions(jobList []*jobsdb.JobT) {
 	if len(processUserIDs) > 0 {
 		userJobsToProcess := make(map[string][]*jobsdb.JobT)
 		userEventsToProcess := make(map[string][]interface{})
+		userToSessionMap := make(map[string]int64)
+
 		logger.Debug("Post Add Processing")
 		proc.Print()
 
 		//We clear the data structure for these users
 		for userID := range processUserIDs {
 			userJobsToProcess[userID] = proc.userJobListMap[userID]
 			userEventsToProcess[userID] = proc.userEventsMap[userID]
+			userToSessionMap[userID] = proc.userToSessionIDMap[userID]
 			delete(proc.userJobListMap, userID)
 			delete(proc.userEventsMap, userID)
```

Comment:
We need to delete this from userToSessionIDMap too.

Reply:
We would need the following two lines to remove the entry from priority queue.

Reply:
not removing here as we want to maintain the session for the user until it has a period of inactivity = sessionThresholdInS.

Clearing here would mean, each time we have a bunch of sessionThresholdEvents, we would assign a new session.

wanted to keep it based on sessionThresholdInS

Reply:
A session for us is every n events or t secs. With the above flow, we are saying a session is always t secs that broken down to batches of n events. 
Do we need it that way? If needed, can't we increase n and make sure everything falls in t sec?

Reply:
My preference would be to keep it simple unless there is a customer use case. Did we need this for Torpedo?

Reply:
since, as far as I could get from the code, we always update the lastTS whenever we receive events for that user.
We remove the user from the list if the time since lastTS of a user > t secs.

Am I understanding it right?